src/Stratis.Bitcoin/Utilities/AsyncExecutionEvent.cs

```diff
@@ -0,0 +1,202 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Stratis.Bitcoin.Utilities
+{
+    /// <summary>
+    /// Asynchronous event handler that can be registered with <see cref="AsyncExecutionEvent{TSender, TArg}/>.
```

Comment:
xml element is not closed, use
`<see cref="AsyncExecutionEvent{TSender,TArg}"/>.`

Reply:
ah tx